### ISSUE TITLE
CR line endings present in international files.

### ISSUE THREAD

**Issue by ryanpcmcquen** [exaile/exaile#42]
When compiling I get this error:

```
./ja.po:1827: warning: internationalized messages should not contain the '\r' escape sequence
./ja.po:1827: warning: internationalized messages should not contain the '\r' escape sequence
./ja.po:1827: warning: internationalized messages should not contain the '\r' escape sequence
./ja.po:1827: warning: internationalized messages should not contain the '\r' escape sequence
```

Strangely enough, running ```sed -i 's,/r,,' *.po``` inside that directory did not fix the error message or change any line endings. Any idea what might be going on?

Thanks!

**Comment by virtuald** [comment]
That's very strange. What are you running to get this error, I don't seem to get it.

**Comment by genodeftest** [comment]
This should be fixed with #31. Do you use the latest git master?

**Comment by ryanpcmcquen** [comment]
No, I was building off the 3.4.3 tarball. No error on master. Thanks!
